frontend/JavaScript/product.js

```diff
@@ -0,0 +1,80 @@
+window.onload = function(){
+    fetch(`http://127.0.0.1:7002/categoryTree`,{
+    method : 'GET',
+    mode :'cors',
+    headers:{
+      'Access-Control-Allow-Origin':'*',
+      'Accept': 'application/json',
+      'Content-Type': 'application/json;charset=utf-8',
+      
+    }
+    
+  }).then(response => response.json()).then(data =>{
+    cat1 = data[0];
```

Comment:
Add Negative use case code block as well . Like if some error is received in the api response , code block to handle it properly should be added .

Reply:
Negative use case code block for error response status is now added.